### PR TITLE
Auth fix sourav

### PR DESCRIPTION
**Pull request by souravahmed1**
Auth Fixes

### CODE REVIEW

File: libs/common/src/lib/logging/logging.model.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { WinstonModule, utilities as nestWinstonModuleUtilities } from 'nest-win
 
 
 export const initWinston = (apiTitle: string) => {
+  console.log(__dirname)
```
Comment (amitkverma) [inline libs/common/src/lib/logging/logging.model.ts]: remove console log from here

File: libs/common/src/lib/security/authenticatedRouteMiddleware.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (amitkverma) [inline libs/common/src/lib/security/authenticatedRouteMiddleware.ts]: extra spaces

File: libs/common/src/lib/security/authenticatedRouteMiddleware.ts [hunk 3]
```diff
@@ -6,7 +8,7 @@ import { IJwtTokenData } from '@spotlyt-backend/data/interfaces';
 
 
 export const Authenticate = createParamDecorator(
-    (roles: unknown, ctx: ExecutionContext) => {
+    (args: unknown, ctx: ExecutionContext) => {
```
Comment (amitkverma) [inline libs/common/src/lib/security/authenticatedRouteMiddleware.ts:9]: Why args are `unknown`? Can we get the types for them

Comment (souravahmed1) [reply libs/common/src/lib/security/authenticatedRouteMiddleware.ts:9]: its not required as of now, cause we are doing authorization via CASL

**Comment by souravahmed1** [comment]
#4 done

**Pull request merged**